addons/ballistics/stringtable.xml

```diff
@@ -2309,6 +2312,7 @@
         </Key>
         <Key ID="STR_ACE_Ballistics_statBulletMass">
             <English>Bullet mass</English>
+            <German>Kugelgewicht</German>
```

Comment:
"Geschossmasse" maybe?
Or "Kugelmasse"
It's not really a ball, so Geschoss fits better IMO.

Reply:
Projektil sagt man im Deutschen

Reply:
Projektilgewicht klingt aber doof. Projektilmasse?

Reply:
würde sagen Projektilgewicht